lib/inferno/dsl/fhir_evaluation/evaluator.rb

```diff
@@ -30,7 +30,9 @@ def evaluate(data, config = Config.new)
           end
 
           Rule.descendants.each do |rule|
-            rule.new.check(context) if active_rules.include?(rule.name.demodulize)
+            next unless active_rules.include?(rule.name.demodulize)
```

Comment:
Rather than:
```ruby
something.each { next if some_criteria? }
```
Use:
```ruby
something.select { some_criteria? }.each { }
```

Reply:
Thanks for the tip. I also found I can make the two loops into one.

Modified:

https://github.com/inferno-framework/inferno-core/pull/672/commits/c28276067b8fb10681da5cf6b56c68df8262bb8c